misc/python/materialize/cli/ci_annotate_errors.py

```diff
@@ -120,6 +120,10 @@
 
 TIMESTAMP_IN_PANIC_RE = re.compile(rb" \d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{6}Z ")
 
+JOURNALCTL_LOG_LINE_RE = re.compile(
```

Comment:
This will reduce annotation spam by grouping same log lines together even when they have a different timestamp.